.eslintrc.json

```diff
@@ -4,14 +4,16 @@
 	"extends": [
 		"wordpress",
 		"plugin:react/recommended",
-		"plugin:jsx-a11y/recommended"
-	],
+		"plugin:jsx-a11y/recommended",
+		"plugin:jest/recommended"
+  ],
 	"env": {
 		"browser": false,
 		"es6": true,
 		"node": true,
-		"mocha": true
-	},
+		"mocha": true,
+		"jest/globals": true
```

Comment:
This doesn't include `before` and `after`, which means we still need `mocha: true`?  I'm a bit surprised by that.